### PR TITLE
dev-1443_disable_slack_in_revengine

### PR DESCRIPTION
**Pull request by gurjittoptal**
Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)
DRAFT 

#### What's this PR do?
Disable slack notifications

#### Why are we doing this? How does it help us?
N/A

#### How should this be manually tested?
Can be but for that we will need to add an organizationslackintegration. Once that is done a successful donation will need to be done. And this won't trigger a slack notification.

#### Have automated unit tests been added?
N/A

#### How should this change be communicated to end users?
N/A

#### Are there any smells or added technical debt to note?
No

#### Has this been documented? If so, where?
N/A

#### What are the relevant tickets?
**(Note: Please use syntax [JIRA-123] to link any relevant tickets)**
[DEV-1443]

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
No

#### Are there next steps? If so, what? Have tickets been opened for them?
**(Note: Please use syntax [JIRA-123] to link any relevant tickets)**

[DEV-1443]: https://news-revenue-hub.atlassian.net/browse/DEV-1443?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/contributions/webhooks.py [hunk 1]
```diff
@@ -6,7 +6,9 @@
 import pytz
 
 from apps.contributions.models import Contribution, ContributionStatus
-from apps.slack.models import SlackNotificationTypes
+
+
+# from apps.slack.models import SlackNotificationTypes
```
Comment (harsha09) [inline apps/contributions/webhooks.py:11]: Please remove the comments

File: apps/contributions/webhooks.py [hunk 2]
```diff
@@ -80,7 +82,7 @@ def handle_payment_intent_succeeded(self):
         # Grab payment_method_id
         contribution.provider_payment_method_id = self.obj_data.get("payment_method")
 
-        contribution.save(slack_notification=SlackNotificationTypes.SUCCESS)
```
Comment (harsha09) [inline apps/contributions/webhooks.py:83]: @x110dc : How about using a configuration variable to enable or disable slack notifications here?
At later point of time slack notification can be enabled if needed, without changing the code again.

**Review by harsha09 — CHANGES_REQUESTED** [review]

File: apps/contributions/webhooks.py (hunk 1 above)
Comment (x110dc) [reply apps/contributions/webhooks.py:11]: Agreed, just remove the line.

File: apps/contributions/webhooks.py (hunk 2 above)
Comment (x110dc) [reply apps/contributions/webhooks.py:83]: I think it's fine disabled this way for now. I opened a ticket to consider a variable to enable/disable: https://news-revenue-hub.atlassian.net/browse/DEV-1544

**Review by x110dc — CHANGES_REQUESTED** [review]
After the comment is removed it will LGTM.

**Review by x110dc — APPROVED** [review]

**Pull request merged**